File: Frontend/angular-app/src/app/modules/inventory/entities/persons-debited-commodities.ts
export interface PersonsDebitedCommodities {
  id: number;
  warehousesTitle: string;
  warehouseId: number;
  assetSerial: string;
  documentDate: string;
  debitReferenceTitle: string;
  price: number;
  depreciationTypeBaseId: number | null;
  assetGroupId: number;
  depreciatedPrice: number;
  unitId: number | null;
  measureId: number | null;
  isActive: boolean;
  commodityId: number;
  commoditySerial: string;
  categoryLevelCode: string;
  categoryTitle: string;
  measureTitle: string;
  searchTerm: string;
  unitsTitle: string;
  depreciationTitle: string;
  assetGroupTitle: string;
  documentNo: number;
  
  documentItemId: number;
  personId: number;
  assetId: number | null;
  debitTypeId: number;
  expierDate: string;
  quantity: number;
  firstName: string;
  lastName: string;
  fatherName: string;
  nationalNumber: string;
  fullName: string;
  accountReferenceId: number;
  commodityTitle: string;
  commodityCode: string;
  isHaveWast: boolean;
  isAsset: boolean;
  isConsumable: boolean;
  documentItemsDescription: string;
  documentDescription: string;
  totalDescription: string;
  
}

File: Backend/Microservices/Inventory/Eefa.Inventory.Domain/Aggregates/PersonsDebitedCommodities/PersonsDebitedCommodities.cs
using Eefa.Common.Data;
using Eefa.Common.Domain;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
namespace Eefa.Inventory.Domain
{
    [Table("PersonsDebitedCommodities", Schema = "inventory")]
    public partial class PersonsDebitedCommodities : DomainBaseEntity, IAggregateRoot
    {
        public int? WarehouseId { get; set; } = default!;
        public int DocumentItemId { get; set; } = default!;
        
        public int? PersonId { get; set; } = default!;
        public int? DebitAccountReferenceId { get; set; } = default!;
        public int? DebitAccountReferenceGroupId { get; set; } = default!;
        
        public int? CommodityId { get; set; } = default!;
        public int? MeasureId { get; set; } = default!;
        public string CommoditySerial { get; set; } = default!;
        public int? AssetId { get; set; } = default!;
        public int? UnitId { get; set; } = default!;
        public int DebitTypeId { get; set; } = default!;
        public DateTime DocumentDate { get; set; } = default!;
        public DateTime ExpierDate { get; set; } = default!;
        public double Quantity { get; set; } = default!;
        public bool IsActive { get; set; } = default!;
        public string Description { get; set; } = default!;
       
    }
   

}
